src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -55,16 +54,6 @@ void regmodule_offline_transformations(py::module m) {
         py::arg("model"),
         py::arg("params_with_custom_types"));
 
-    m_offline_transformations.def(
```

Comment:
@alexsu52, I hope we don't use it in NNCF. Please confirm.

Reply:
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/145ebcd8-db74-4b63-917a-8944704e9370)